electron-builder.yml

```diff
@@ -12,11 +12,6 @@ files:
     - node_modules/**/*
     - package.json
 
-extraResources:
-  - from: 'build/saturn/l2node-${platform}-${arch}'
-    to: 'saturn-l2-node'
-    filter: ['*']
-
```

Comment:
❗ 

We need to change the way how we are packaging the app. At the moment, the electron-builder puts all node_modules files into an ASAR archive. At runtime, Electron presumably hooks into the Node.js module loader so that JavaScript files can be loaded from the ASAR archive.

Before your PR, this `extraResources` section told the electron-builder to package additional files into the app bundle. When we wanted to execute Saturn L2, we would find the binary in the file tree.

When I package the app produced by your PR, it cannot start Station Core:

<img width="632" alt="Screenshot 2023-05-03 at 16 01 43" src="https://user-images.githubusercontent.com/1140553/235939705-d35ed70c-9965-4b8e-9a18-75ad88c8ca84.png">

There are different ways how we can solve this problem.
1. You can disable ASAR, so that the app bundle contains all JS files directly in the filesystem.
2. Maybe you could add `@filecoin/core` to `extraResources` to make it available for spawning a new Node.js process. That may require changing the layout of node_modules so that all `@filecoin/core ` modules are nested in its directory or maybe installing `@filecoin/core` in a different place. Maybe instead of using `npm install @station/core`, we can download `@station/core` source files into `build/station-core` and then run `npm install` inside that directory?
3. _(I am sure we can find more options.)_


**How to reproduce the problem:**

```bash
# 1. Package the app for distribution: 
❯ npm run package

# 2. Run the packaged app: 
❯ ./dist/mac/Filecoin\ Station.app/Contents/MacOS/Filecoin\ Station
```

Reply:
This is a very important find!

I'll give both options a try 🙏

Reply:
btw, an easier way to open an app from the terminal is

```bash
$ open dist/mac/Filecoin\ Station.app

```

Reply:
I tried to get it to work while keeping `asar`, but there was issue after issue. Disabled it now and can confirm it works